src/input_cp2k_dft.F

```diff
@@ -9721,4 +9729,143 @@ SUBROUTINE create_cdft_opt_section(section)
+                          description="Define the parameters for PWDFT+U+V calculations", &
+                          n_keywords=0, &
+                          n_subsections=1, &
+                          repeats=.FALSE.)
+      NULLIFY (keyword)
+      CALL keyword_create(keyword, __LOCATION__, &
+                          name="_SECTION_PARAMETERS_", &
+                          description="Controls the activation of the PWDFT+U+V calculations", &
+                          usage="&DFT_U_PLUS_V ON", &
+                          default_l_val=.FALSE., &
+                          lone_keyword_l_val=.TRUE.)
+      CALL section_add_keyword(section, keyword)
+      CALL keyword_release(keyword)
+
+      NULLIFY (subsection)
+      CALL section_create(subsection, __LOCATION__, &
+                          name="LINK", &
+                          description="parameters describing which atom pair and their respective atomic "// &
+                          "levels should be included in the DFT+U+V calculations. "// &
+                          "NOTE : it is only supported by the PWDFT method", &
+                          n_keywords=5, &
+                          n_subsections=0, &
+                          repeats=.TRUE.)
+
+      CALL keyword_create(keyword, __LOCATION__, &
+                          name="ATOM_PAIR", &
+                          description="atom pair contributing the DFT+U+V calculations", &
+                          usage="ATOM_PAIR 1 2", &
+                          n_var=2, &
+                          type_of_var=integer_t)
+      CALL section_add_keyword(subsection, keyword)
+      CALL keyword_release(keyword)
+
+      CALL keyword_create(keyword, __LOCATION__, &
+                          name="ATOMIC_LEVELS", &
+                          description="atomic level of each atomic orbital contributing the DFT+U+V calculations", &
+                          usage="ATOMIC_LEVELS 1 2", &
+                          n_var=2, &
+                          type_of_var=integer_t)
+      CALL section_add_keyword(subsection, keyword)
+      CALL keyword_release(keyword)
+
+      CALL keyword_create(keyword, __LOCATION__, &
+                          name="ANGULAR_MOMENTUMS", &
+                          description="angular momentum of each atomic orbital contributing the DFT+U+V calculations", &
+                          usage="ATOMIC_LEVELS 1 2", &
+                          n_var=2, &
+                          type_of_var=integer_t)
+      CALL section_add_keyword(subsection, keyword)
+      CALL keyword_release(keyword)
+
+      CALL keyword_create(keyword, __LOCATION__, &
+                          name="V", &
+                          description="Effective parameter V (in eV)", &
+                          repeats=.FALSE., &
+                          n_var=1, &
+                          type_of_var=real_t, &
+                          default_r_val=0.0_dp, &
+                          unit_str="au_e", &
```

Comment:
Here the unit is `au_e`, which is good because CP2K uses almost exclusively atomic units for its inputs. However, in the description above it says `eV`.

Reply:
thanks will fix it.

Reply:
should be fixed now